### PR TITLE
Fix the build files so that prow job can run successfully

### PR DESCRIPTION
**Pull request by brahmaroutu**

### CODE REVIEW

**Review by wlan0 — APPROVED** [review]

**Comment by wlan0** [comment]
LGTM

File: deploy/base/rbac.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@ kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: objectstorage-controller-role
+  namespace: default
```
Comment (NicolasT) [inline deploy/base/rbac.yaml:6]: It's odd this needs to be added, and not expected. Could you elaborate why these were added? `kustomize` (or `kubectl -k`) should take care of these already.

File: deploy/base/rbac.yaml [hunk 2]
```diff
@@ -25,7 +26,8 @@ rules:
 kind: ClusterRoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: system:objectstorage-controller
+  name: objectstorage-controller
+  namespace: default
```
Comment (NicolasT) [inline deploy/base/rbac.yaml:30]: A `ClusterRoleBinding` doesn't have a `namespace`.

File: deploy/base/rbac.yaml [hunk 3]
```diff
@@ -34,6 +36,7 @@ metadata:
 subjects:
   - kind: ServiceAccount
     name: objectstorage-controller-sa
+    namespace: default
```
Comment (NicolasT) [inline deploy/base/rbac.yaml:39]: Same question as https://github.com/kubernetes-sigs/container-object-storage-interface-controller/pull/30/files#r553022404

File: deploy/base/rbac.yaml [hunk 4]
```diff
@@ -43,6 +46,7 @@ kind: Role
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: objectstorage-controller
+  namespace: default
```
Comment (NicolasT) [inline deploy/base/rbac.yaml:49]: Same question as https://github.com/kubernetes-sigs/container-object-storage-interface-controller/pull/30/files#r553022404

File: deploy/base/rbac.yaml [hunk 5]
```diff
@@ -57,6 +61,7 @@ kind: RoleBinding
...
```
Comment (NicolasT) [inline deploy/base/rbac.yaml:64]: Same question as https://github.com/kubernetes-sigs/container-object-storage-interface-controller/pull/30/files#r553022404

File: deploy/base/rbac.yaml [hunk 6]
```diff
@@ -65,6 +70,7 @@ metadata:
 subjects:
   - kind: ServiceAccount
     name: objectstorage-controller-sa
+    namespace: default
```
Comment (NicolasT) [inline deploy/base/rbac.yaml:73]: Same question as https://github.com/kubernetes-sigs/container-object-storage-interface-controller/pull/30/files#r553022404

**Pull request merged**